### DISCUSSION TITLE
Add Forgotten Souls to Ghosts list for Slayer tasks

### DISCUSSION THREAD

**Discussion by brasil221** [runelite/runelite#16423] [Ideas]
Plugin does not highlight Forgotten Souls while a player is on a Ghost Slayer task.

Expected behavior: Plugin highlights Forgotten Souls while a player is on a Ghost Slayer task.

**Comment by Nightfirecat** [comment]
Created an issue in #16425 to track this enhancement

**Reply by brasil221** [reply]
Oh, thank you!! I couldn't figure out how to create it as an issue or bug report or anything; this was the only way I could find to post. First time contributing, so sorry if I did it wrong but thanks for elevating it for me! :)
